pkg/assembler/backends/inmem/certifyVuln_test.go

```diff
@@ -355,6 +355,89 @@ func TestIngestCertifyVulnerability(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name:   "Query No Vuln - with novuln boolen",
+			InVuln: []*model.VulnerabilityInputSpec{noVulnInput},
+			InPkg:  []*model.PkgInputSpec{p2},
+			Calls: []call{
+				{
+					Pkg:  p2,
+					Vuln: noVulnInput,
+					CertifyVuln: &model.ScanMetadataInput{
+						Collector:      "test collector",
+						Origin:         "test origin",
+						ScannerVersion: "v1.0.0",
+						ScannerURI:     "test scanner uri",
+						DbVersion:      "2023.01.01",
+						DbURI:          "test db uri",
+						TimeScanned:    t1,
+					},
+				},
```

Comment:
can add another input here as well to test that it doesn't tkae the additional?

Reply:
added! Only returns the novuln nodes